src/popup/router/routes.ts

```diff
@@ -216,11 +217,11 @@ export const routes: WalletAppRouteConfig[] = [
       },
 
       {
-        path: 'details/transactions/:hash',
+        path: 'details/transactions/:hash/:transactionOwner',
```

Comment:
You would need to add `backRoute: { name: ROUTE_MULTISIG_DETAILS_TRANSACTIONS},` into `meta` property in order to fix back button issue.